<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImplementingStandardEditor</class>
 <widget class="QWidget" name="ImplementingStandardEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>检测标准：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="standardEidtBtn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="MyComboBox" name="standardSelectBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>表格名称：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="tableEditBtn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QComboBox" name="tableSelectBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>限值类别：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="classEditBtn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QComboBox" name="classSelectBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>检测项目：</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="addTestItemBtn">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="standardEditWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>标准名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="standerNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>标准号：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="standardNumEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="standardEditOK">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="tableEditWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>表格名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="tableNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>检测类型：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="tableEditOk">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="MyComboBox" name="testTypeBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="classEditWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>类别名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="classNameEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="classEditOK">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="itemEditWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>高限：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>项目：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="itemEditOK">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="tradeEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>低限：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>行业：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>单位：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="UnitEditBox" name="unitBox"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="higherEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lowerEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="MyComboBox" name="testItemSelectBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>UnitEditBox</class>
   <extends>QComboBox</extends>
   <header location="global">uniteditbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
